.github/workflows/integration.yml

```diff
@@ -151,21 +151,28 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
-
+      - name: Get dbt-core version
```

Comment:
Would this work for when the event is a push and not a PR? It looks like we only account for PR events and not push/schedule events. We keep the branch names the same between repos so could we do something where we just default to use the PR branch or the push branch? Something like...

if (pull request event)
   dbt-version = GITHUB_BASE_REF # env variable that GitHub provides
else 
   dbt-version = GITHUB_REF # env variable that GitHub provides

Reply:
this would probably break if you ever chain pull requests (create PR to a non-release branch) though I agree a strict improvement over what exists today

Reply:
@McKnight-42 let's use the logic leah described in this PR. We can chat about other options on the table after this is merged